resources/js/electron-builder.js

```diff
@@ -61,53 +59,69 @@ if (isBuilding) {
+        } else {
+            // As we can't copy into a subdirectory of ourself we need to copy to a temp directory
+            let tmpDir = mkdtempSync(join(os.tmpdir(), 'nativephp'));
+
+            console.warn('=====================');
+            console.warn('* * * INSECURE BUILD * * *');
+            console.warn('Secure app bundle not found! Building with exposed source files.');
+            console.warn('See https://nativephp.com/docs/publishing/building#security for more details');
+            console.warn('=====================');
+
+            copySync(process.env.APP_PATH, tmpDir, {
+                overwrite: true,
+                dereference: true,
+                filter: (src, dest) => {
+                    let skip = [
+                        // Skip .git and Dev directories
+                        join(process.env.APP_PATH, '.git'),
+                        join(process.env.APP_PATH, 'docker'),
+                        join(process.env.APP_PATH, 'packages'),
+
+                        // Only needed for local testing
+                        join(process.env.APP_PATH, 'vendor', 'nativephp', 'electron', 'vendor'),
+                        join(process.env.APP_PATH, 'vendor', 'nativephp', 'laravel', 'vendor'),
+
+                        join(process.env.APP_PATH, 'vendor', 'nativephp', 'php-bin'),
+                        join(process.env.APP_PATH, 'vendor', 'nativephp', 'electron', 'bin'),
+                        join(process.env.APP_PATH, 'vendor', 'nativephp', 'electron', 'resources'),
+                        join(process.env.APP_PATH, 'node_modules'),
+                        join(process.env.APP_PATH, 'dist'),
+                        join(process.env.APP_PATH, 'build'),
+
+                        join(process.env.APP_PATH, 'storage', 'framework'),
+                        join(process.env.APP_PATH, 'storage', 'logs'),
+                    ];
+
+                    let shouldSkip = false;
+                    skip.forEach((path) => {
+                        if (src.indexOf(path) === 0) {
+                            shouldSkip = true;
+                        }
+                    });
+
+                    return !shouldSkip;
+                }
+            });
+
+            copySync(tmpDir, appPath);
+
+            // Electron build removes empty folders, so we have to create dummy files
+            // dotfiles unfortunately don't work.
+            writeJsonSync(join(appPath, 'storage', 'framework', 'cache', '_native.json'), {})
+            writeJsonSync(join(appPath, 'storage', 'framework', 'sessions', '_native.json'), {})
+            writeJsonSync(join(appPath, 'storage', 'framework', 'testing', '_native.json'), {})
+            writeJsonSync(join(appPath, 'storage', 'framework', 'views', '_native.json'), {})
+            writeJsonSync(join(appPath, 'storage', 'app', 'public', '_native.json'), {})
+            writeJsonSync(join(appPath, 'storage', 'logs', '_native.json'), {})
```

Comment:
It's better to use `outputJsonSync` to create any missing directory in path or add additionall call `ensureDirSync`, but I prefer `outputJsonSync`